data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -121,6 +122,17 @@
       <enable_snapshots config:type="boolean">false</enable_snapshots>
       <initialize config:type="boolean">true</initialize>
       <partitions config:type="list">
+	% if ($check_var->('IPXE_UEFI', '1')) {
+        <partition>
+          <create config:type="boolean">true</create>
+          <filesystem config:type="symbol">vfat</filesystem>
+          <format config:type="boolean">true</format>
+	  <partition_id t="integer">259</partition_id>
+	  <mount>/boot/efi</mount>
+          <mountby config:type="symbol">uuid</mountby>
+          <size>512M</size>
+        </partition>
+	% }
```

Comment:
Do you have to partition manually ?

Reply:
What do you mean about "partition manually"? Are you saying that skipping the entire <partitions> to ask SLE install wizard to partition automatically?

Reply:
Yes.

Reply:
I tried two ways:
1. There are two existing partitions in the autoyast template, without this new "EFI partiton", system will fail to boot.
2. I tried automatical partitioning by removing the existing two partitions, but "/var/lib/libvirt/image" was not mounted on a seperate partition. It was not inline with manual installation.